release/x86_64-freebsd10/Vagrantfile

```diff
@@ -81,6 +81,9 @@ Vagrant.configure(2) do |config|
   #   sudo apt-get install -y apache2
   # SHELL
   config.vm.provision "shell", inline: <<-SHELL
-    pkg install -y bash gmake ruby rubygem-bundler git-subversion python2
+    freebsd-update --not-running-from-cron fetch install
+    pkg upgrade -y
```

Comment:
None. The `firstboot-freebsd-update` updates the system, but the `firstboot-pkgs` installs a list of packages on the first boot, and for this box, the list is empty:

https://github.com/freebsd/freebsd/blob/release/10.3.0/release/tools/vagrant.conf#L8

At the time of this writing, this is the upgrade list:

    xkeyboard-config: 2.16 -> 2.17
    python27: 2.7.11_1 -> 2.7.12
    pixman: 0.32.8 -> 0.34.0
    perl5: 5.20.3_8 -> 5.20.3_13
    pciids: 20151224 -> 20160621
    libglesv2: 11.0.8 -> 11.2.2
    libglapi: 11.0.8 -> 11.2.2
    libedit: 3.1.20150325_1 -> 3.1.20150325_2,1
    libdrm: 2.4.65_1,1 -> 2.4.66,1
    libGL: 11.0.8 -> 11.2.2
    libEGL: 11.0.8 -> 11.2.2
    gettext-runtime: 0.19.6 -> 0.19.8.1
    gbm: 11.0.8 -> 11.2.2
    freetype2: 2.6.2 -> 2.6.3
    expat: 2.1.0_3 -> 2.1.1_2
    dri: 11.0.8,2 -> 11.2.2,2